CodeGeneration/Sources/generate-swift-syntax/templates/swiftsyntax/TokenKindFile.swift

```diff
@@ -204,6 +204,17 @@ let tokenKindFile = SourceFileSyntax(leadingTrivia: copyrightHeader) {
     }
   }
 
+  try! ExtensionDeclSyntax("private extension RawTokenKind") {
```

Comment:
Simple `precondition` statement was evaluating slowly. This extension helped solving this issue. 🕵🏻

Reply:
That makes sense. Thank you!